docs/manual/inference.tex

```diff
@@ -316,6 +316,14 @@
   is motivated by
   \ahref{https://github.com/gradle/gradle/issues/1319}{Gradle issue \#1319}.
 
+\item[-s]
+  If this flag is present, then projects which are not buildable---for which
```

Comment:
Nit: The rest of the manual puts a space around em-dashes, so this should do so as well, for consistency.